com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorderSettings.cs

```diff
@@ -91,9 +91,12 @@ static string GenerateBindingId()
         /// <returns>Transform associated with binding ID</returns>
         static Transform GetBinding(string id)
         {
-            // return BindingManager.Get(m_BindingId) as GameObject;
+#if UNITY_2020_1_OR_NEWER
```

Comment:
@benoithudson Do we know when BindingManager was made public so using reflection was no longer necessary? Is 2020.1 the appropriate year to check against in this case?

Reply:
I don't, but as Viktoria mentions I think it would be the `com.unity.recorder` version not the Unity version. We can bump the required version (in `package.json`) for recorder to whatever shipped with Unity 2021.1 and it should have the API.

Reply:
Check with the recorder team (Marie I guess?) what version would be most appropriate. Probably 2.5.something